packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -698,36 +694,28 @@ export abstract class FluidDataStoreContext
 
 	/**
 	 * After GC has run, called to notify the data store of routes used in it. These are used for the following:
-	 *
 	 * 1. To identify if this data store is being referenced in the document or not.
-	 *
 	 * 2. To determine if it needs to re-summarize in case used routes changed since last summary.
-	 *
-	 * 3. These are added to the summary generated by the data store.
-	 *
-	 * 4. To notify child contexts of their used routes. This is done immediately if the data store is loaded.
-	 * Else, it is done when realizing the data store.
-	 *
-	 * 5. To update the timestamp when this data store or any children are marked as unreferenced.
+	 * 3. To notify child contexts of their used routes. This is done immediately if the data store is loaded.
+	 * Else, it is done by the data stores's summarizer node when child summarizer nodes are created.
 	 *
 	 * @param usedRoutes - The routes that are used in this data store.
 	 */
 	public updateUsedRoutes(usedRoutes: string[]) {
 		// Update the used routes in this data store's summarizer node.
 		this.summarizerNode.updateUsedRoutes(usedRoutes);
 
-		/**
-		 * Store the used routes to update the channel if the data store is not loaded yet. If the used routes changed
-		 * since the previous run, the data store will be loaded during summarize since the used state changed. So, it's
-		 * safe to only store the last used routes.
-		 */
-		this.lastUsedRoutes = usedRoutes;
-
-		// If we are loaded, call the channel so it can update the used routes of the child contexts.
-		// If we are not loaded, we will update this when we are realized.
-		if (this.loaded) {
-			this.updateChannelUsedRoutes();
+		// If the channel doesn't exist yet (data store is not realized), the summarizer node will update it
+		// when it creates child nodes.
+		if (!this.channel) {
```

Comment:
Side note: We should switch `this.loaded` to be a property that checks if `this.channel !== undefined`.  This file is split between people checking `loaded` v. `channel`, you just switched in this PR between the two.